### ISSUE TITLE
Error in Compiling: You can't use ``\relax' after \the. ^^I^^I\end{tabu}`

### ISSUE THREAD

**Issue by Sabicool** [HarveySheppard/yLaTeX#13]
Tried to recompile some files from a while ago and comes up with these errors.

```
You can't use `\relax' after \the. ^^I^^I\end{tabu}
You can't use `\relax' after \the. ^^I}
```

I moved the hyperref package to the end of the preamble because I thought it was being overwritten by something else but this did not resolve the issue.
```
\usepackage{hyperref}
\hypersetup{
	pdfcreator=LaTeX,
	colorlinks=true,
	linkcolor=blue,
	linktoc=all,
	urlcolor=blue,
	citecolor=blue,
	filecolor=blue
}
```

**Comment by HarveySheppard** [comment]
Hey @Sabicool,

Can you specify which class you are having troubles with?

Please note that tabu is now deprecated, so you may indeed run into troubles. I believe the LaTeX3 project is doing some minimal maintenance so not sure about this, but you newer version of the package might be causing compatibility issues...

**Comment by Sabicool** [comment]
the `yReport` class. I shall use an earlier version of the tabu package and I'll put in the result

**Comment by HarveySheppard** [comment]
Please note that I removed the tabu code from yReport and included booktabs instead in commit d387bd84ae0d2ce111d3755a5743127fb46a0faf (made today).

So changing from tabu to booktabs should fix all issues. It will also change the design. Most notably, you will loose alternated colors in rows and colored lines. All those functionalities can be reproduced using booktabs I think...
